### DISCUSSION TITLE
Remote Docker Interpreter will not initialize

### DISCUSSION THREAD

**Discussion by jayguidos** [Camelcade/Perl5-IDEA#2479] [Q&A] [answered]
We have 4 developers all using the same versions of OS, IntelliJ, and the latest Perl plugins.

We have all tried to install a remote Docker interpreter, and 3 times this has worked perfectly. We all pick the same Docker image, and on the 3 that work properly we get a popup that asks to pick a Perl5 Executable file, and the path looks like this:

perlhost:///usr/local/bin/perl

This appears to be a path inside the docker image so we pick it and the interpeter initializes properly.  

The other problematic DEV machine goes through the same process, but instead of getting prompted for a perlhost:/// path it looks like its trying to pick a file on the local machine:

/home/jacy.jones

The OK button remains dimmed no matter what we pick, and all we can do is cancel, and the install fails.

I noticed that on the sucessful installs you are spinning up a docker container during the install. This also happens on the failed install, but something else is going wrong and we cannot complete the install. 

Can you suggest something to help us fix this?

J

**Comment by hurricup** [comment, marked as answer]
Well, this need to be troubleshooted. The first thing I would do - to find the difference between machines:
- Os, os version
- docker version
- IDE type and version
- Enabled plugins set for IDE

The process is pretty straightforward, IDE starts the container and introspecting its file system with pseudoe FS (perlhost:/)
Feels like on failing machine users choose 'Add local perl interpreter' instead 'Add docker perl interpreter'

**Reply by jayguidos** [reply]
We could not see why this was not working - but this developer is a plugin junkie and had many plugins installed.  We reinstalled his entire IntelliJ with only standard plugins and now the remote interpreter gets set up correctly. Thanks for your suggestions.
